components/url_loader_component/url_loader_component.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_URL_LOADER_COMPONENT_URL_LOADER_COMPONENT_H_
+#define BRAVE_COMPONENTS_URL_LOADER_COMPONENT_URL_LOADER_COMPONENT_H_
+
+#include <list>
+#include <map>
+#include <memory>
+#include <string>
+
+#include "base/callback.h"
+#include "base/memory/weak_ptr.h"
+#include "net/traffic_annotation/network_traffic_annotation.h"
+#include "url/gurl.h"
+
+namespace network {
+class SharedURLLoaderFactory;
+class SimpleURLLoader;
+}  // namespace network
+
+namespace url_loader_component {
```

Comment:
I would try to craft a better name. `component` already has several different meanings, so this class is not really a "component". Also it is manages multiple loaders, not one. Maybe `APIRequestHelper` or something

Reply:
Renamed